contracts/src/interfaces/lseth/IRiverV1.sol

```diff
@@ -0,0 +1,279 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.20;
+
+import { ISharesManagerV1 } from "./ISharesManagerV1.sol";
+
+/// @title River Interface (v1)
+/// @author Kiln
+/// @notice The main system interface
+interface IRiverV1 is ISharesManagerV1 {
```

Comment:
We should consider trimming down the interfaces for integrations. We can remove 90% of these functions since most are unused in the integration. Same for the other integrations.